src/webcomponents/clinical/clinical-analysis-update.js

```diff
@@ -117,7 +117,7 @@ export default class ClinicalAnalysisUpdate extends LitElement {
     }
 
     opencgaSessionObserver() {
-        this.users = OpencgaCatalogUtils.getUsers(this.opencgaSession.study);
+        this._users = OpencgaCatalogUtils.getUsers(this.opencgaSession.study);
```

Comment:
Please make sure `this._users` has been properly initialized in the `init` function.